src/styles/Button.module.scss

```diff
@@ -1,42 +1,49 @@
 @import './Constants.scss';
 
+@mixin disableDefaultButtonStyles {
+  font-family: inherit;
+  background: none;
+  border: none;
+  padding: 0;
+}
+
+$hover-transition-time: 0.2s;
+$click-transition-time: 0.01s;
+$button-height: 3.25rem;
+
```

Comment:
Zmienne są po to żeby z nich korzystać - w tym wypadku *-transition-time których kilkukrotnie używamy, oraz button-height, który przyda nam się do obliczenia border-radiusa niżej w kodzie